### PR TITLE
Two new packages: zxcvbn and babel-polyfill

### PR DESCRIPTION
**Pull request by binarykitchen**
Both externs written by hand and tested.

### CODE REVIEW

File: babel-polyfill/build.boot [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  pom {:project     'cljsjs/babel-polyfill
+       :version     +version+
+       :description "Provides polyfills necessary for a full ES2015+ environment."
+       :url         "https://github.com/babel/babel/tree/master/packages/babel-polyfill"
+       :scm         {:url "https://github.com/SMX-LTD/cljsjs-packages"}
+       :license     {"MIT" "https://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+    (download :url "https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.20.0/polyfill.js")
```
Comment (Deraen) [inline babel-polyfill/build.boot]: Please use versioned url to download the archive or the files. Use the `+lib-version+` var you have defined to build the url:

```clj
(def +lib-version+ "1.2.3")

...

(download :url (format "https://github.com/example/project/archive/%s.zip" +lib-version+)
          :unzip true)
```

If the library for some reason doesn't use git tags and doesn't have versioned downloads, you can use git sha to download a specific version. Use git sha for the commit which released the version.

```clj
(def +lib-version+ "1.2.3")
(def +git-sha+ "97ffda1f0e1d45fde4dcfbfcef0fcd904c714918")

...

(download :url (format "https://github.com/example/project/archive/%s.zip" +git-sha+)
          :unzip true)
```

Using versioned downloads ensures that the builds are reproductible and that the CI version will use the correct archive/files to build and deploy the package.

Comment (binarykitchen) [inline babel-polyfill/build.boot]: @Deraen understand, but i am afraid, not even those files exist on their repo. have a look
https://github.com/babel/babel/tree/master/packages/babel-polyfill

there are no polyfill.js and no polyfill.min.js

Comment (Deraen) [inline babel-polyfill/build.boot]: The CDN has versioned urls: https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.20.0/polyfill.js

**Comment by binarykitchen** [comment]
@Deraen oh i get it ... here you go, fixed with versions in URLs for babel-polyfill from CDN

**Pull request merged**